language/functional_tests/tests/testsuite/genesis/genesis_validator_set.mvir

```diff
@@ -0,0 +1,25 @@
+import 0x0.LibraAccount;
+import 0x0.ValidatorSet;
+
+// check that the genesis validator set has been created and has the expected size
+main() {
+    assert(LibraAccount.exists(0x1D8), 77);
+    assert(ValidatorSet.size() == 10, 78);
+
+    // check that one of the addresses we expect is in there...
```

Comment:
@runtian-zhou here is your test

Reply:
Yea I was just thinking about adding some random proptest. I can add that test once we merge this.

Reply:
What would a proptest buy us here?

Reply:
We can implement a round trip test: Generate a validator set at random, create a genesis from that writeset, and decode such writeset to make sure we can decode the exact same information as the inputted validator set.

Reply:
I see the value of doing that once for a specific validator set, but (IMO) trying random combinations of validator sets is unlikely to produce interesting bugs that a single one won't. If desired, we can implement tests like this, but we'll need to fix the serialization/deserialization of `types/validator_set` first (it was written before the implementation of `Vector`) was completed.